bulkExport/export-script.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import boto3
+import re
+from awsglue.transforms import *
+from awsglue.utils import getResolvedOptions
+from pyspark.context import SparkContext
+from awsglue.context import GlueContext
+from awsglue.job import Job
+from awsglue.dynamicframe import DynamicFrame
+from datetime import datetime
+
```

Comment:
Add a comment explaining what this file is doing and how it is used with glue and what functionality it is providing

Think of adding the information a forker of this repo will need to determine to keep this file or not for their deployment

Reply:
updated

Reply:
updated